src/OrbitBase/NotFoundOrTest.cpp

```diff
@@ -14,6 +15,34 @@
 
 namespace orbit_base {
 
+TEST(NotFoundOr, Construct) {
```

Comment:
Maybe 6 (or 3 if we also wish to test assignment) separate test cases would be better?

Reply:
Same question about the test for `CanceledOr`

Reply:
So I split the tests. And renamed them a bit. The name `Construct` was also wrong in a sense. Because I didn't wanna test constructing the type. The whole point is that construction and assignment is borrowed from `std::variant`. The things that need to be tested are the getters.